cypress/e2e/workTray.feature

```diff
@@ -37,9 +37,9 @@ Feature: As an internal Hackney User eg. HO, AHM
     And process status
 
 # This scenario will be fully working once the BE filter ticket is completed
-   Scenario: AC3 - Worktray filter display by default
-     Given I am on the MMH home page
-     Then I can see my worktray dashboard
+  #  Scenario: AC3 - Worktray filter display by default
```

Comment:
What is the reason for this to be commented out? Do you think we can fix this before we release the patch edit?